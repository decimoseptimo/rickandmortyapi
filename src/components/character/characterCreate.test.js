import React from "react"
import renderer from "react-test-renderer"

import CharacterCreate from "./characterCreate"

it("renders correctly", () => {
  const tree = renderer
    .create(<CharacterCreate onSubmit={_ => {}} onClose={_ => {}} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
