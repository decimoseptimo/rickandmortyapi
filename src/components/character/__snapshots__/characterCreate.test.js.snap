// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-0"
>
  <h2
    className="chakra-heading css-1drbkg9"
  >
    Create Character
  </h2>
  <div
    className="chakra-form-control css-1kxonj9"
    role="group"
    spacing={4}
  >
    <form
      onSubmit={[Function]}
    >
      <div
        className="chakra-stack css-6vyap7"
      >
        <div
          className="css-0"
        >
          <label
            className="chakra-form__label css-uvho5g"
            htmlFor="CharacterForm"
            id="CharacterForm-label"
          >
            Name
            <span
              aria-hidden={true}
              className="chakra-form__required-indicator css-0"
              role="presentation"
            >
              *
            </span>
          </label>
          <input
            aria-required={true}
            className="chakra-input css-0"
            disabled={false}
            id="Name"
            name="name"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Name"
            readOnly={false}
            required={true}
          />
        </div>
        <div
          className="css-0"
        >
          <label
            className="chakra-form__label css-uvho5g"
            htmlFor="CharacterForm"
            id="CharacterForm-label"
          >
            Image
            <span
              aria-hidden={true}
              className="chakra-form__required-indicator css-0"
              role="presentation"
            >
              *
            </span>
          </label>
          <input
            aria-required={true}
            className="chakra-input css-0"
            disabled={false}
            id="Image"
            name="image"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Image"
            readOnly={false}
            required={true}
          />
        </div>
        <div
          className="css-0"
        >
          <label
            className="chakra-form__label css-uvho5g"
            htmlFor="CharacterForm"
            id="CharacterForm-label"
          >
            Status
            <span
              aria-hidden={true}
              className="chakra-form__required-indicator css-0"
              role="presentation"
            >
              *
            </span>
          </label>
          <input
            aria-required={true}
            className="chakra-input css-0"
            disabled={false}
            id="Status"
            name="status"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Status"
            readOnly={false}
            required={true}
          />
        </div>
        <div
          className="css-0"
        >
          <label
            className="chakra-form__label css-uvho5g"
            htmlFor="CharacterForm"
            id="CharacterForm-label"
          >
            Gender
            <span
              aria-hidden={true}
              className="chakra-form__required-indicator css-0"
              role="presentation"
            >
              *
            </span>
          </label>
          <input
            aria-required={true}
            className="chakra-input css-0"
            disabled={false}
            id="Gender"
            name="gender"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Gender"
            readOnly={false}
            required={true}
          />
        </div>
        <div
          className="css-0"
        >
          <label
            className="chakra-form__label css-uvho5g"
            htmlFor="CharacterForm"
            id="CharacterForm-label"
          >
            Origin
            <span
              aria-hidden={true}
              className="chakra-form__required-indicator css-0"
              role="presentation"
            >
              *
            </span>
          </label>
          <input
            aria-required={true}
            className="chakra-input css-0"
            disabled={false}
            id="Origin"
            name="origin"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Origin"
            readOnly={false}
            required={true}
          />
        </div>
        <div
          className="css-0"
        >
          <label
            className="chakra-form__label css-uvho5g"
            htmlFor="CharacterForm"
            id="CharacterForm-label"
          >
            Location
            <span
              aria-hidden={true}
              className="chakra-form__required-indicator css-0"
              role="presentation"
            >
              *
            </span>
          </label>
          <input
            aria-required={true}
            className="chakra-input css-0"
            disabled={false}
            id="Location"
            name="location"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Location"
            readOnly={false}
            required={true}
          />
        </div>
      </div>
      <div
        className="chakra-stack css-15k61gf"
      >
        <button
          className="chakra-button css-oqfhsh"
          onClick={[Function]}
          type="button"
        >
          Cancel
        </button>
        <button
          className="chakra-button css-oqfhsh"
          type="submit"
        >
          Save
        </button>
      </div>
    </form>
  </div>
</div>
`;
